chart/values.yaml

```
@@ -514,6 +514,7 @@ workers:
 
   # Select certain nodes for airflow worker pods.
   nodeSelector: {}
+  priorityClassName: ""
```

```suggestion
  priorityClassName: ~
```

to be consistent, preference @jedcunningham ?

---

Yeah 👍

---

@jedcunningham , if I have it as tilde(~), then the validation fails, thoughts

```
Processing file:  chart/values.yaml
None is not of type 'string'

Failed validating 'type' in schema['properties']['workers']['properties']['priorityClassName']:
    {'default': '',
     'description': 'Set priority value from the priority class name',
     'type': 'string'}

On instance['workers']['priorityClassName']:
    None
None is not equal to the default of 

Failed validating 'default' in schema['properties']['workers']['properties']['priorityClassName']:
    {'default': '',
     'description': 'Set priority value from the priority class name',
     'type': 'string'}

On instance['workers']['priorityClassName']:
    None
```

---

You just need to change the type in the schema, e.g: https://github.com/apache/airflow/blob/0ac3b8c3dd749c59e60cf0169580b9e7c5049d9e/chart/values.schema.json#L431-L434